input/spec.md

```diff
@@ -29,9 +29,9 @@ Keys are as follows:
 * version="1.0"
 * name : string
 * CVE : list of string-refs
-* Responsibly_disclosed : list of bool-refs
+* Coordinated_disclosure : string ("true", "false", or "unknown")
 * Categories : list of strings from : 'kernel' (a kernel vulnerability), 'signature' (errors processing signatures on APKs), 'system' (a vulnerability in the system libraries or processes), 'network' (a vulnerability exploitable by a network attacker), 'permissions' (incorrect permissions on a file), 'app' (exploitable app with elevated permissions)
-* Severity : string
+* Severity : string (__Deprecated feature__ - do not use on new submissions and may be removed in the future)
```

Comment:
On further consideration we may want to use "Severity" to implement the "critical or not" analysis we used last time round and then compare it with the more robust analysis possible using the more detailed data you have collected this time around.

Reply:
OK - we presumably need just a critical/not critical flag for this then?
(Currently there are only 5 vulnerabilities which actually have a severity rating, and 4 of these are rated as `uncertain`)

Reply:
Yes. Currently it defaults to "critical" and non-critical ones have severity ratings specified but we should put a flag on all of them.